### DISCUSSION TITLE
The icon needs to be changed in order to resemble a prism and not a pyramid

### DISCUSSION THREAD

**Discussion by joel-wlf** [prisma/prisma#22590] [Feedback]
It has has come to my notice, that the Icon used by the prism database project in fact resembles a pyramid and not a prism. Please correct according to mathematical guidelines.

See: https://testbook.com/maths/difference-between-prism-and-pyramid

**Comment by gerkim62** [comment]
this is a funny one 😊

**Comment by joel-wlf** [comment]
After all, it is a very serious matter.

**Comment by SevInf** [comment]
I believe the only way for us to get out this is to change company name and all associated products to Pyramid.

Some advantages:
- Changing a text in roughly 10 000 000 places (napkin estimate) is easy enough even I can do it, while designing a new logo requires an actual skill.
- If we put an actual prism on our logo, people will inevitably confuse us with Pink Floyd and ask to perform their favourite songs. And trust me, my rendition of "Great Gig In The Sky" will not bring happiness to your life.  Neither, I think, Roger Waters would appreciate all of your Postgres questions.
- Years later, when we retire, we can tell people that we've built Pyramid when we were young and seem way cooler than we actually are.

Disadvantages:
- None as far as I can see.

**Reply by favna** [reply]
Calling yourself pyramid also comes with the advantage of being able to make silly Illuminati jokes

**Comment by DaGutsche** [comment]
honestly it is important, imagine apples logo being a pear

**Reply by aqrln** [reply]
That's just Pear Company in iCarly.

**Comment by gerkim62** [comment]
I don't think they should change this  logo, it looks cute already. If you are not satisfied with it, why don't you create a simple browser extension that changes the icon for every prisma website you visit, together with favicon?

**Comment by daniel31x13** [comment]
Any updates here? This is so crucial otherwise I’m gonna have to move to other alternatives for my projects.

Edit: Yes, I’m joking, and I love this project!
